src/ECharts/Theme.js

```diff
@@ -1,8 +1,12 @@
-require("echarts/theme/dark")
-require("echarts/theme/infographic")
-require("echarts/theme/macarons")
-require("echarts/theme/roma")
-require("echarts/theme/shine")
-require("echarts/theme/vintage")
+"use strict";
 
-exports.forceExport = null
+exports._dark = { name: "dark", value: require("echarts/theme/dark") };
+exports._infographic = { name: "infographic", value: require("echarts/theme/infographic") };
+exports._macarons = { name: "macarons", value: require("echarts/theme/macarons") };
+exports._roma = { name: "roma", value: require("echarts/theme/roma") };
+exports._shine = { name: "shine", value: require("echarts/theme/shine") };
+exports._vintage = { name: "vintage", value: require("echarts/theme/vintage") };
```

Comment:
These `value`s are actually `undefined`, since the requires export nothing, but unless the bundler is super clever it shouldn't be obvious that these things are unused/undefined.